tests/expectations/test_generate_diagnostic_checklist.py

```diff
@@ -30,9 +30,9 @@ def test_print_diagnostic_checklist__first_iteration():
 
 
 def test_print_diagnostic_checklist__second_iteration():
-    output_message = (
-        ExpectColumnValuesToEqualThree__SecondIteration().print_diagnostic_checklist()
-    )
+    output_message = ExpectColumnValuesToEqualThree__SecondIteration(
+        column="values"
```

Comment:
Feels iffy but all these diagnostic methods are instance methods - I have a ticket to try and make them classmethods so don't need to pass meaningless params but that's less of a priority atm